docs/examples/notebooks/learn/TestStatistics.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>Explain in words that you're going to study a single channel with two samples, signal and background. The signal expected event rate is 10 events, and the background expected event weight is 100 events, with an assigned 10% uncertainty on the background.</p><p> </p><p>"Therefore, observing 100 events is more background-like and observing 110 events is more signal-like."</p><p> </p><p>We want to try to help build up intuition about the test statistics from the start of model definitions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scikit-hep/pyhf/pull/1014/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>